### ISSUE TITLE
Fonts inconstant across Operating Systems

### ISSUE THREAD

**Issue by Ben-Ryder** [Ben-Ryder/NEA#15]
**Describe the bug**
The games global system font: `constants.FONTS["main"] ("Arial" currently)` appears different on Windows 10 than on xubuntu (my main dev environment)

**To Reproduce**
Run game on different Operating System (ie Windows 10). Not tested on mac. 

**Expected behavior**
Text to look consistent throughout Operating Systems 

**Potential Fix**
Include font files within the project, in an `NEA/data/fonts` folder.
Then import and use these so the project isn't OS dependent on fonts, and looks consistent,

**Comment by Ben-Ryder** [comment]
Found `Arial `is not the font used, but found pygame uses `pygame.font.match_font()`, which is why it isn't reverting to built-in font. In windows `Arial` must be located and used as the actual font, **not tested though.**
Using this I found on xubuntu the font used is `FreeSans`, but have no idea if this is available on all OS. So will continue to look into using font files, as it will give freedom and also better customisation.
